<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.Hcl.DirectCommerce.Orders_with_Additional_Pay_methods_gift_coupon_cert.Place_a_phone_order_for_C1_item_with_Gift_Card_pay_method_Fully_Paid" tests="1" time="315.549" ignored="0" failures="1" timestamp="15 May 2020 06:06:39 GMT" skipped="0" hostname="IN2-1025604LT" errors="0">
  <testcase name="C1_OA" time="315.549" classname="com.Hcl.DirectCommerce.Orders_with_Additional_Pay_methods_gift_coupon_cert.Place_a_phone_order_for_C1_item_with_Gift_Card_pay_method_Fully_Paid">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at com.Hcl.DirectCommerce.uiActions.customerInformation.clickOnEndOrder(customerInformation.java:203)
	at com.Hcl.DirectCommerce.Orders_with_Additional_Pay_methods_gift_coupon_cert.Place_a_phone_order_for_C1_item_with_Gift_Card_pay_method_Fully_Paid.C1_OA(Place_a_phone_order_for_C1_item_with_Gift_Card_pay_method_Fully_Paid.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- C1_OA -->
</testsuite> <!-- com.Hcl.DirectCommerce.Orders_with_Additional_Pay_methods_gift_coupon_cert.Place_a_phone_order_for_C1_item_with_Gift_Card_pay_method_Fully_Paid -->
